core-bundle/src/Command/MigrateCommand.php

```diff
@@ -278,6 +277,8 @@ static function ($hash) use ($commandsByHash) {
                 return false;
             }
         }
+
+        throw new \RuntimeException('This should never happen');
```

Comment:
Won't PHPSTan report here? 🙃

Reply:
It did not on my local system. 🤷‍♂

Reply:
Funny. I remember it did while the developers meeting with unreachable statement 🤷‍♂

Reply:
That was a different file. 😄

Reply:
But it seems to work also in the pipeline.